crates/ty_python_semantic/src/types/instance.rs

```diff
@@ -69,6 +66,13 @@ pub struct NominalInstanceType<'db> {
 }
 
 impl<'db> NominalInstanceType<'db> {
+    fn from_class(class: ClassType<'db>) -> Self {
```

Comment:
```suggestion
    // Keep this field private, so that the only way of constructing `NominalInstanceType` instances
    // is through the `Type::instance` constructor function.
    fn from_class(class: ClassType<'db>) -> Self {
```

Reply:
Going to change this to "Keep this method" etc